### PR TITLE
Fixed typo in __all__

### PR DESCRIPTION
**Pull request by 3lnc**
Idk if we need such test at all.

### CODE REVIEW

File: test_elasticsearch_dsl/test_aggs.py [hunk 1]
```diff
@@ -166,3 +166,7 @@ def test_filters_correctly_identifies_the_hash():
         }
     } == a.to_dict()
     assert a.filters.group_a == query.Q('term', group='a')
+
+def test_import__all__():
+    for name in aggs.__all__:
+        exec('from elasticsearch_dsl.aggs import {0}'.format(name))
```
Comment (HonzaKral) [inline test_elasticsearch_dsl/test_aggs.py]: I think `assert hasattr(aggs, name)` would be better here, `exec` is scary even in tests :)

**Comment by HonzaKral** [comment]
I think tests are great but I left a comment on the implmentation.

Thanks for another PR!

File: test_elasticsearch_dsl/test_aggs.py (hunk 1 above)
Comment (3lnc) [inline test_elasticsearch_dsl/test_aggs.py]: Indeed its scary )
Damn, haven't figured out such a simple way to test. TY, fixed.

**Comment by HonzaKral** [comment]
Thank you!

**Pull request merged**